src/components/main/fruits/fruitsListSection/__test__/FruitList.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { mockFruitListFromApi } from "../../../../mocks";
+import FruitList from "../FruitList";
+import { render, screen } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
+import { HashRouter } from "react-router-dom";
+
+describe("FruitList section calls API services", () => {
```

Comment:
Is this the container? Is nos visible from inside the test, plz change the description

Reply:
Also, if not the container, it does not have any api or services call, please change description